meerk40t/gui/scene/scene.py

```diff
@@ -278,13 +278,15 @@ def on_right_mouse_up(self, event):
 
     def on_magnify_mouse(self, event):
         """
-        Magnify Mouse is a Mac Event called with pinch to zoom on a trackpad.
+        Magnify Mouse is a Mac-only Event called with pinch to zoom on a trackpad.
         """
-        magnify = event.GetMagnification()
-        if magnify > 0:
-            self.scene.event(event.GetPosition(), "zoom-in")
-        if magnify < 0:
-            self.scene.event(event.GetPosition(), "zoom-out")
+        magnification = event.GetMagnification()
+        if event.IsGestureStart():
+            self.scene.event(event.GetPosition(), "gesture-start")
+        elif event.IsGestureEnd():
+            self.scene.event(event.GetPosition(), "gesture-end")
+        else:
```

Comment:
The gesture-start gesture-end stuff might not be needed but could be of some use at some point.